flybywire-aircraft-a320-neo/effects/LIGHT_A32NX_WingLarge.fx

```diff
@@ -22,7 +22,7 @@ Drag=0.00, 0.00
 X Particle Velocity=0.00, 0.00
 Y Particle Velocity=0.00, 0.00
 Z Particle Velocity=0.00, 0.00
-X Rotation=0.00, 0.00
+X Rotation=20.00, 20.00
```

Comment:
Just a note. This rotation applies to both wing lights as they use the same .fx file. It is fine in most cases, but I would prefer to have this information in the `systems.cfg` instead so the light knows nothing about its orientation.